### DISCUSSION TITLE
Suricata CVE error

### DISCUSSION THREAD

**Discussion by archangelneo18** [telekom-security/tpotce#943] [Q&A]
Did something happen to the Surata CVE dash? its not populating any of the CVE information and getting this error message.
![Screen Shot 2021-11-19 at 4 06 13 PM](https://user-images.githubusercontent.com/18698637/142691897-1715c1dd-da9e-4d99-9ae7-b0962437c687.png)

**Comment by jacobholtz** [comment]
I have the same issue after a fresh install on the latest version

**Comment by Ganesh-403** [comment]
@archangelneo18 
From what I can tell, there isn't a known, general T-Pot bug that causes the **Suricata CVE dashboard** to stop populating. The discussion doesn't include the exact error message, so it's difficult to determine the root cause.

A few things I'd check first:

1. **Verify that Suricata events are still being ingested**
   - Open **Discover** in Kibana and confirm that Suricata (`eve.json`) events are arriving.
   - If there are no recent Suricata documents, the dashboard won't have data to enrich or visualize. T-Pot's troubleshooting guide recommends checking container status and logs first. :contentReference[oaicite:0]{index=0}

2. **Check the relevant container logs**
   ```bash
   dps
   docker logs -f suricata
   docker logs -f logstash
   docker logs -f elasticsearch
   docker logs -f kibana
   ```
   Also review:
   ```bash
   cat ~/tpotce/data/tpotinit.log
   ```
   These can help identify ingestion, parsing, or Elasticsearch indexing errors. :contentReference[oaicite:1]{index=1}

3. **Determine whether the issue is only the dashboard**
   If Suricata alerts are visible in Discover but the CVE visualization is empty, then the problem is likely with:
   - the enrichment pipeline,
   - the index pattern,
   - or the visualization itself,
   rather than Suricata detection.

4. **Check for Elasticsearch or mapping errors**
   If the screenshot showed a Kibana query or aggregation error, it could indicate:
   - missing fields,
   - changed mappings after an upgrade,
   - or an incompatible dashboard/index pattern.

At this point, a few additional details would help narrow it down:

- Which **T-Pot version** are you running?
- Which **Suricata version** is installed?
- Does the problem occur after an upgrade or on a fresh installation?
- Could you share the **full error message** from the dashboard (or a screenshot), rather than just the empty visualization?

Those details will make it much easier to determine whether this is an indexing issue, a dashboard compatibility issue, or a problem with the enrichment pipeline.

If this solves your problem, feel free to mark it as the accepted answer so others can find it easily.
